docs/source/linchpincli_linchpin_invgen.rst

```diff
@@ -1,24 +0,0 @@
-Invgen
```

Comment:
Can we just comment this out for now? It will likely be used in the future to generate inventories from existing topologies.